CfgLoadouts.hpp

```diff
@@ -69,10 +71,18 @@ class CfgLoadouts {
   // US: HK416 - Multi-Terrain Pattern "Loadouts\us_hk416_mtp_apex.hpp" (apex)
   // US: HK416 - Tropical "Loadouts\us_hk416_trop_apex.hpp" (apex)
   // US: M4 - Gray/Green "Loadouts\us_m4_ucp.hpp"
+  // US: M4 - Gray/Green "Loadouts\blumsv_us_m4_ucp.hpp" (msv)
   // US: M4 - Tan "Loadouts\us_m4_ocp.hpp"
+  // US: M4 - Tan "Loadouts\blumsv_us_m4_ocp.hpp" (msv)
   // US: M16 - M81 "Loadouts\us_m16_m81.hpp"
+  // US: M16 - Marine Pattern Desert "Loadouts\usmc_m16_marpatd.hpp"
```

Comment:
Is this supposed to be blumsv?

Reply:
No, that's for the standard loadout. The EMC version is on line 78. Going to be changing all the paths from blumsv to emc right now.